public/components/CampaignInformationEdit/CampaignInformationEdit.js

```diff
@@ -5,22 +5,14 @@ import {formatMillisecondDate} from '../../util/dateFormatter';
 
 class CampaignInformationEdit extends React.Component {
 
-  state = {
-    isCampaignDirty: false
-  }
+  static propTypes = {
+    updateCampaign: PropTypes.func.isRequired,
+    markDirty: PropTypes.func.isRequired
```

Comment:
is markDirty required? I can't see it being called without updateCampaign (so can't updateCampaign just markDirty)